src/drivers/actuators/voxl_esc/qc_esc_packet.c

```diff
@@ -151,6 +153,36 @@ int32_t qc_esc_create_rpm_packet4_fb(int16_t rpm0, int16_t rpm1, int16_t rpm2, i
 	return qc_esc_create_packet(ESC_PACKET_TYPE_RPM_CMD, (uint8_t *) & (data[0]), 10, out, out_size);
 }
 
+int32_t qc_esc_create_rpm_div2_packet4_fb(int32_t rpm0, int32_t rpm1, int32_t rpm2, int32_t rpm3,
+				     uint8_t led0, uint8_t led1, uint8_t led2, uint8_t led3, 
+					 int32_t fb_id, uint8_t *out, uint16_t out_size)
+{
+	int16_t data[5];
+	uint16_t leds = 0;
+	
+	if (fb_id != -1) { fb_id = fb_id % 4; }
```

Comment:
There is a bunch of copied code in here. It very similar to qc_esc_create_rpm_packet4_fb. Can you make a common function to do the common stuff that both of these can call to avoid that?

Reply:
This is a good idea, I will make it happen